src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  isActive?: boolean | false,
```

Comment:
why did you add false here?
```suggestion
  isActive?: boolean,
```